### DISCUSSION TITLE
aria-label on Button and Link components

### DISCUSSION THREAD

**Discussion by aroyan** [chakra-ui/chakra-ui#5899] [Q&A] [answered]
Is there any way to add aria-label to Button and Link components? I have 3 icons and wrap it in Link component but I got an warning on lighthouse to put an aria-label to Link and Button. Or there is an alternative components to fix this? Thank you

**Comment by eckertalex** [comment, marked as answer]
You can simply add the aria-label as a prop to the component. If you have a button that is just an icon, I suggest you use Chakra's `IconButton` as its types will remind you to add a `aria-label`. Here is a [Codesandbox](https://codesandbox.io/s/chakra-discussion-aria-label-byycvp?file=%2Fsrc%2Fapp.tsx)

**Reply by aroyan** [reply]
Thank you @eckertalex . Just figured it out and yes, it's solved my problems
